markdown/util.py

```diff
@@ -85,19 +85,7 @@
 # Extensions should use "markdown.util.etree" instead of "etree" (or do `from
 # markdown.util import etree`).  Do not import it by yourself.
 
-try:  # pragma: no cover
-    # Is the C implementation of ElementTree available?
-    import xml.etree.cElementTree as etree
-    from xml.etree.ElementTree import Comment
-    # Serializers (including ours) test with non-c Comment
-    etree.test_comment = Comment
```

Comment:
The [failing test][1] is caused by removing this line. The fix would be to update both the [tests][2] and the [serializer][3] to both use `from xml.etree.ElementTree.Comment` without the extra custom `test_comment` object.

[1]: https://travis-ci.org/Python-Markdown/markdown/jobs/644817928#L247
[2]: https://github.com/Python-Markdown/markdown/blob/master/tests/test_apis.py#L472-L515
[3]: https://github.com/Python-Markdown/markdown/blob/master/markdown/serializers.py#L45-L48